ddcommon/Cargo.toml

```diff
@@ -55,6 +58,7 @@ hyper-rustls = { version = "0.27", default-features = false, features = [
     "tls12",
     "aws-lc-rs",
 ] }
+memfd = { version = "0.6" }
 
```

Comment:
I just checked `memfd` crate and it uses `rustix` as dependency - which means this doesn't bring new libc dependencies unnecessarily 👍 LGTM: